drltrace/drltrace.c

```diff
@@ -332,7 +318,7 @@ options_init(client_id_t id)
                 USAGE_CHECK(res == 1, "invalid -verbose number");
             }
         } else {
-            NOTIFY("UNRECOGNIZED OPTION: \"%s\""NL, token);
+            VNOTIFY(1, "UNRECOGNIZED OPTION: \"%s\""NL, token);
```

Comment:
Shouldn't this be NOTIFY?  It's a fatal error and this will tell the user which option was invalid, right?